### ISSUE TITLE
Sync bookmark keywords from Firefox

### ISSUE THREAD

**Issue by st3iny** [floccusaddon/floccus#1675]
### Describe the feature you'd like to request

In Firefox, a user might create keywords for search fields/engines by right-clicking them. This works on search bars of almost all sites including Google, GitHub, Amazon, Wikipedia etc. Then, a user may type the keyword in the address bar followed by a search term and will automatically be directed to the search result page as if he or she used the search bar directly on the corresponding website.

Let me make an example:
1. Go to youtube.com
2. Create a keyword bookmark of the main search at the top and assign the keyword `@yt`.
3. Create a new tab and type `@yt rickroll` and hit enter.
4. You are navigated to https://www.youtube.com/results?search_query=rickroll skipping the landing page of YouTube entirely.

Internally, this works by creating a bookmark with a special keyword in Firefox. A user may assign or change the keyword at any time by editing said bookmark. So essentially, those quick search actions are just regular bookmarks with a special keyword property.

It would be **super awesome** if Floccus could sync those keywords across multiple devices. Although, I'm not sure if it is even possible to somehow access those keywords from an extension.

PS: I'm sorry if this is a duplicate. I tried to search for a similar request without luck.

### Describe the solution you'd like

Sync keywords of Firefox bookmarks.

### Describe alternatives you've considered

Not sync the keywords via Floccus. Instead, set them manually on each device.

**Comment by huyz** [comment]
Keywords for those searching for this:
- Firefox: Search Shortcuts
- Chrome: Search Engines, Custom Search Engines, Site Search

**Comment by marcelklehr** [comment]
Good idea! I think this is not possible as firefox does not expose this via their APIs. :(
